### PR TITLE
Bump istio api

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

**Review by howardjohn — CHANGES_REQUESTED** [review]
I think you are masking a real bug. We should either wait for the fix (sound slike https://github.com/istio/api/pull/1283 will do so?) or revert the breakage in api

**Comment by hzxuzhonghu** [comment]
I have no strong feeling, will wait for that merged, and try it here

**Comment by hzxuzhonghu** [comment]
Try the newest api, and reverted the test skip.

File: go.sum [hunk 1]
```diff
@@ -936,6 +936,8 @@ honnef.co/go/tools v0.0.1-2019.2.3/go.mod h1:a3bituU0lyd329TUQxRnasdCoJDkEUEAqEt
 istio.io/api v0.0.0-20190515205759-982e5c3888c6/go.mod h1:hhLFQmpHia8zgaM37vb2ml9iS5NfNfqZGRt1pS9aVEo=
 istio.io/api v0.0.0-20200201001607-7b905a0a6e49 h1:dA52YJ/YsM3cSIB0maN7S4RvFTKz/9gDA0/FrN/59xE=
 istio.io/api v0.0.0-20200201001607-7b905a0a6e49/go.mod h1:bcY3prusO/6vA6zGHz4PNG2v79clPyTw06Xx3fprJSQ=
+istio.io/api v0.0.0-20200212140015-6fa980ccf401 h1:xUzNk8ttD9lj04TBWQyCruzz+SEs2aZmqgf726MXBjA=
+istio.io/api v0.0.0-20200212140015-6fa980ccf401/go.mod h1:bcY3prusO/6vA6zGHz4PNG2v79clPyTw06Xx3fprJSQ=
```
Comment (howardjohn) [inline go.sum]: you will likely need to run `go mod tidy` or some test will fail

**Review by howardjohn — APPROVED** [review]

File: go.sum (hunk 1 above)
Comment (hzxuzhonghu) [reply go.sum]: ok

**Pull request closed without merging**